tests/wpt/metadata/dom/events/ProgressEvent.html.ini

```diff
@@ -3,3 +3,4 @@
   bug: https://github.com/servo/servo/issues/10740
   [document.createEvent() should not work with ProgressEvent.]
     expected: FAIL
```

Comment:
There's nothing magic about `bug` annotations, and also nothing that removes unused sections like this.